### PR TITLE
Simplify Parameter documentation for Set-DnsServerCache

### PR DESCRIPTION
**Pull request by rdbahm**
It appears that the documentation for this cmdlet was largely carried over from the inner workings of how the values are stored in the system registry - since this is documentation on how to use the PowerShell cmdlet, I have clarified the parameters.

Specifically, I've altered the description of the TimeSpan parameters and the StoreEmptyAuthenticationResponse parameter, all of which focused on the hex values internally stored by the DNS Server instead of the values that need to be passed to the cmdlet.

### CODE REVIEW

**Comment by e0i** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

@get-itips @tseward

Peer review needed for this PR. Thanks.

File: docset/windows/dnsserver/set-dnsservercache.md [hunk 1]
```diff
@@ -183,9 +183,9 @@ Accept wildcard characters: False
 ```
 
 ### -MaxTtl
-Specifies how many seconds (0x0-0xFFFFFFFF) a record is saved in cache.
-If you use the 0x0 setting, the DNS server does not cache records.
-The default setting is 0x15180 (86,400 seconds, or one day).
+Specifies how long (0 to 4294967294 seconds) a record is saved in cache.
+If the timespan is set to 0 seconds, the DNS server does not cache records.
```
Comment (nenonix) [inline docset/windows/dnsserver/set-dnsservercache.md]:
```suggestion
If the TimeSpan is set to 0 seconds, the DNS server does not cache records.
```

**Comment by get-itips** [comment]
Hello @rdbahm 
Thanks for your contribution.
I tested this, and the parameters only accept TimeSpan objects, either provided inline or as a variable:

image

image

image

I couldn't use the ranges that you include in your pull request as a parameter for MaxNegativeTtl and MaxTtl.
Could you please correct this in your PR?

Thanks again!

**Comment by rdbahm** [comment]
@get-itips The values I provided in MaxNegativeTtl and MaxTtl were converted from the hex values in the original documentation - but you're correct, the cmdlet itself doesn't seem to accept this value. Experimentally, I was able to determine that the actual maximum value appears to be 2592000 seconds (30 days) for both on Server 2016. I'll make the change for both of those parameters.

**Comment by get-itips** [comment]
Would you mind adding an example to the Example list in your Pull Request to enhance your explanation of the parameters?
Something like 

`Set-DnsServerCache -MaxTTL 02.00:00:00 -MaxNegativeTtl 00.00:20:00`

> This command sets the MaxTTL to 2 days and the MaxNegativeTtl to 20 minutes

**Comment by rdbahm** [comment]
Okay, I've added the example as requested and clarified the MaxTtl and MaxNegativeTtl parameters.

**Comment by get-itips** [comment]
Ok thanks let's see @tseward comments on this.

**Comment by tseward** [comment]
Looks good, thanks.

**Comment by e0i** [comment]
@AndreaBarr The PR can be merged. Thanks.

**Pull request merged**